cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1127,6 +1151,13 @@
           ]
         }
       ]
+    },
+    "UseHyperthreading": {
```

Comment:
I'd rename this condition to `DisableHyperthreading`

Reply:
Done